### DISCUSSION TITLE
Welcome to astrology-api Discussions!

### DISCUSSION THREAD

**Discussion by ryuphi** [ryuphi/astrology-api#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bsmsaturn** [comment]
shall I get the position numbers and lord details for the planet in response?
